### PR TITLE
docs(*) update streams and service mesh to 1.1

### PR DESCRIPTION
**Pull request by bungle**
### Summary

Updates Streams and Service Mesh docs with changes from Kong `1.1`.

### CODE REVIEW

File: app/1.1.x/streams-and-service-mesh.md [hunk 1]
```diff
@@ -5,28 +5,29 @@ title: Streams and Service Mesh
...
-Kong `0.15.0` / `1.0.0` added capability to proxy and route raw `tcp` streams and
-deploy Kong using a service mesh sidecar pattern with mutual `tls` between Kong
-nodes. This tutorial walks you trough a basic setup of a simplified Service Mesh
-deployment using simple tooling: two servers, talking with each other via
-two Kong nodes, in a single host. It introduces new concepts, configuration
-settings and tools along the way. In a production environments almost all
-of this should be automated, it is nice to know how things work on a lower
-level. If you are interested in running Service Mesh with Kubernetes,
-please head over to our [our Kubernetes and Service Mesh example repo](https://github.com/Kong/kong-mesh-dist-kubernetes).
+Kong `0.15.0` / `1.0.0` added capability to proxy and route raw `tcp` and `tls`
```
Comment (h3xar0n) [inline app/1.1.x/streams-and-service-mesh.md]:
```suggestion
Kong `0.15.0` / `1.0.0` added the capability to proxy and route raw `tcp` and `tls`
```

File: app/1.1.x/streams-and-service-mesh.md [hunk 2]
```diff
@@ -5,28 +5,29 @@ title: Streams and Service Mesh
...
+streams and deploy Kong using a service mesh sidecar pattern with mutual
```
Comment (h3xar0n) [inline app/1.1.x/streams-and-service-mesh.md]:
```suggestion
streams and deploy Kong using a service-mesh sidecar pattern with mutual
```

File: app/1.1.x/streams-and-service-mesh.md [hunk 3]
```diff
@@ -5,28 +5,29 @@ title: Streams and Service Mesh
...
+`tls` between Kong nodes. This tutorial walks you trough a basic setup of
+a simplified Service Mesh deployment using simple tooling: two servers,
+talking with each other via two Kong nodes, in a single host. It introduces
+new concepts, configuration settings and tools along the way. In a production
+environments almost all of this should be automated. It is still nice to
```
Comment (h3xar0n) [inline app/1.1.x/streams-and-service-mesh.md]:
```suggestion
environment, almost all of this should be automated. It is still nice to
```

File: app/1.1.x/streams-and-service-mesh.md [hunk 4]
```diff
@@ -293,43 +310,40 @@ supported by default.
...
-as we do in this walkthrough with **Kong B** and **Service B**, the
-network traffic that you used to route directly to **Service B**
-will now be routed to **Kong B**. In this case the **Kong B** knows
-that it is a sidecar for **Service B**, so instead of proxying
-to a Kong service, we override the service endpoint using
-the `origins` configuration on this node only. Without this
-you would end up **Kong B** proxying to **Kong B** loop. Interesting
-side-note to this is that this allows you to change protocols,
-such as `https` to `http` and `tls` to `tcp`.
+**Note**: In a real world you should use `nginx worker user` to make the exceptions
```
Comment (h3xar0n) [inline app/1.1.x/streams-and-service-mesh.md]:
```suggestion
**Note**: In the real world, you should use `nginx worker user` to make the exceptions
```

**Comment by kikito** [comment]
Same as in the other pr: please make it against `master`.

**Pull request merged**